migrations/1595584859678-RemoveStudentDeletePermissionForTeachers.js

```diff
@@ -0,0 +1,53 @@
+const mongoose = require('mongoose');
+// eslint-disable-next-line no-unused-vars
+const { info, error } = require('../src/logger');
+
+const { connect, close } = require('../src/utils/database');
+
+const RoleModel = mongoose.model('role20202707', new mongoose.Schema({
+    name: { type: String, required: true },
+    permissions: [{ type: String }],
+}, {
+    timestamps: true,
+}), 'roles');
+
+const permission = 'STUDENT_DELETE';
+
+// How to use more than one schema per collection on mongodb
+// https://stackoverflow.com/questions/14453864/use-more-than-one-schema-per-collection-on-mongodb
+
+module.exports = {
+    up: async function up() {
+        await connect();
+        // ////////////////////////////////////////////////////
+        // Make changes to the database here.
+        // Hint: Access models via this('modelName'), not an imported model to have
+        // access to the correct database connection. Otherwise Mongoose calls never return.
```

Comment:
Codacy found an issue: [Expected indentation of 2 tabs but found 8 spaces.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5914670)